docker/servicesmw/servicesmw.template.yaml

```diff
@@ -0,0 +1,178 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: mediawiki-servicesmw-${SERVICES_INSTANCE}
```

Comment:
as discussed on Slack - `SERVICES_INSTANCE` is not needed. It should be called the same in both K8s cluster, as this is the naming we follow in all k8s apps.